### PR TITLE
[TACHYON-1414] Group - Show group info in UI and CLI

### PR DESCRIPTION
**Pull request by sundapeng**
Please review:
https://tachyon.atlassian.net/browse/TACHYON-1414

### CODE REVIEW

**Comment by apc999** [comment]
+1 LGTM

**Comment by sundapeng** [comment]
Thank @apc999 for your review, I will rebase this pr once https://github.com/amplab/tachyon/pull/2326 is merged.

**Comment by sundapeng** [comment]
Hi @apc999 , Thank you for your review.
Since https://github.com/amplab/tachyon/pull/2326 have been merged, I have rebased this pr.

**Comment by apc999** [comment]
+1 LGTM. thanks for the improvement!

File: integration-tests/src/test/java/tachyon/shell/TfsShellTest.java [hunk 1]
```diff
@@ -876,35 +886,37 @@ private boolean isInMemoryTest(String path) throws IOException, TachyonException
   public void lsWildcardTest() throws IOException, TachyonException {
     // clear the loginUser
     Whitebox.setInternalState(LoginUser.class, "sLoginUser", (String) null);
-
+    MasterContext.getConf().set(Constants.SECURITY_GROUP_MAPPING,
+        IdentityUserGroupsMapping.class.getName());
+    // TODO
```
Comment (yupeng9) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: What's this TODO for?

File: shell/src/main/java/tachyon/shell/command/AbstractLsCommand.java [hunk 2]
```diff
@@ -46,7 +46,7 @@ protected AbstractLsCommand(TachyonConf conf, TachyonFileSystem tfs) {
    */
   protected void ls(TachyonURI path, boolean recursive) throws IOException {
     List<FileInfo> files = listStatusSortedByIncreasingCreationTime(path);
-    String format = "%-10s%-25s%-15s%-15s%-5s%n";
+    String format = "%-10s%-25s%-15s%-15s%-15s%-5s%n";
```
Comment (yupeng9) [inline shell/src/main/java/tachyon/shell/command/AbstractLsCommand.java]: Could you extract this as a constant to be used here and in test? Also, it'll be good to have some comments about the components.

**Comment by yupeng9** [comment]
Thanks for the improvement. I left some comments.

File: integration-tests/src/test/java/tachyon/shell/TfsShellTest.java (hunk 1 above)
Comment (sundapeng) [inline integration-tests/src/test/java/tachyon/shell/TfsShellTest.java]: Thanks, I will fix it.

File: shell/src/main/java/tachyon/shell/command/AbstractLsCommand.java (hunk 2 above)
Comment (sundapeng) [inline shell/src/main/java/tachyon/shell/command/AbstractLsCommand.java]: Good suggestion, I will fix it.

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**